app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
+      Array.wrap(self[Solrizer.solr_name('title')])
+    end
+
+    def all_descriptions
```

Comment:
Did you first try to name this `description`, thereby overriding CC's method of the same name? Or is there value in having both methods defined on Solr documents?